test/fuzz/gen.ml

```diff
@@ -214,58 +226,77 @@ let rec expr ~block_type ~stack ~locals env =
       in
       drops @ adds
   else
+    let expr_available_2_i32 = expr_available_2_i32 env in
+    let expr_available_2_i64_i32 = expr_available_2_i64_i32 env in
+    let expr_available_2_f32_i32 = expr_available_2_f32_i32 env in
+    let expr_available_2_f64_i32 = expr_available_2_f64_i32 env in
+    let expr_available_3_i32 = expr_available_3_i32 env in
+    let expr_available_1_i64 = expr_available_1_i64 env in
+    let expr_available_1_f32 = expr_available_1_f32 env in
+    let expr_available_1_f64 = expr_available_1_f64 env in
+    let expr_always_available =
+      expr_always_available block loop expr ~locals ~stack env
+    in
```

Comment:
I would simply remove all of this. (Except maybe the `expr_always_available` one)

Reply:
I also removed `expr_always_available` ;)